website/docs/specification/DigitalIdentityAnchor.md

```diff
@@ -0,0 +1,242 @@
+
+![Digital Identity Anchor Concept](DigitalIdentityAnchor.png)
+
+The outcome is that the subject of the DIA (eg the VAT registered business) can prove that they are the registered identity to any other party. In the UNTP context the DIA provides assurance that a DPP (or DCC/DFR/DTE) issuer really is who they say they are. The verification workflow is as follows
+
+* A verifier (eg buyer of an identified product) discovers a DPP for the product and verifies the credential - confirming that the DPP has not been tampered-with, is genuinely issued by party identified by the issuer DID.
+* The DID is resolvable to the DID document which contains a link to the DIA in the DID document `service` end point.
+* Verify the DIA credential and confirm that the DPP issuer DID is contained in the `verifiedDIDList` of the DIA.  
+* Confirm that the issuer did:web of the DIA (the authoritative register) is on the white list of trust anchors.  
+
+The DIA can also be used for similar trust anchoring purposes such as:
+
+* Accreditation authorities issue DIA to assert that a conformity assessment body is accredited against a given scheme.  
+* IP Offices issue DIA to assert that a registered party is the genuine owner of a trademark. 
+* Land registers issue DIA to assert that a regulated party is the owner of a geo-located property.
+
+
+## Requirements
+
+The digital identity anchor is designed to meet the following detailed requirements as well as the more general [UNTP Requirements(https://uncefact.github.io/spec-untp/docs/about/Requirements)]
+
+| ID | Name  | Requirement Statement   | Solution Mapping  |
+| ------ | ---- | --------- | ---------- |
+| DIA-01 | DID Verification | The DIA issuer (registrar) SHOULD confirm that the registered member (subject) is the legitimate holder of a DID before issuing a DIA credential so that the registrar is protected against members falsely claiming ownership of well known DIDs| MAY use the [DID Auth](https://w3c-ccg.github.io/vp-request-spec/#did-authentication) protocol for this purpose. |
+| DIA-02 | DIA Issuer DID | The DIA issuer MUST use did:web (or a trusted variant such as did:tdw) as the DIA issuer and the web domain MUST match the well known domain of the issuing authority so that verifiers can confirm authority identity via public web records.| DIA issuer specification|
+| DIA-03 | Scheme registration | The DIA issuing authority SHOULD register the identity scheme (including the trusted issuer DIDs) with the UN/CEFACT identifier scheme registry so that verifiers can leverage UN maintained scheme metadata to simplify DIA discovery and verification.| See UNTP [Identity Resolver](IdentityResolver.md)|
+| DIA-04 | Subject DID List | The DIA MUST allow multiple subject DIDs to be linked to the same registered identity so long as proof of ownership is verified for each - so that members can rotate multiple active DIDs | `VerfiedDIDList` property|
```

Comment:
```suggestion
| DIA-04 | Subject DID List | The DIA MUST allow multiple subject DIDs to be linked to the same registered identity so long as proof of ownership is verified for each - so that members can rotate multiple active DIDs | `verfiedDIDList` property|
```

Reply:
I`m not a fan of the `verifiedDIDList` concept, it implies the holder would have to engage in multiple did auths with the same DIA issuer. I would rather start with a `credentialSubject.id` did and explore the need for `verifiedDIDList` with some tangible use case.

Reply:
removed the DIDList in the latest commit.